app/models/pawn.rb

```diff
@@ -1,10 +1,38 @@
 class Pawn < Piece
-  
   def self.get_image(color)
     if color == "white"
       return "chess_piece_pawn_white.png"
     elsif color == "black"
       return "chess_piece_pawn_black.png"
     end
   end
+
+  def valid_move?(x,y)
+    @object = self.game.pieces.where(position_x: x, position_y: y)[0]
```

Comment:
lets think about `@object.color == "white"` and `@object.color == "black"` for a second.. what those methods want to convey is if a piece its my own or not.. maybe this can be done in a less explicit way.. for example, we could add a method

```
def own_piece?(other_piece)
# checks if my own color is the same as other piece's color
end
```

this is another way to make the code more concise and readable